src/App.css

```diff
@@ -1,22 +1,73 @@
-.App {
+
+.app {
   text-align: center;
+  display: flex;
+  height: 90%;
 }
 
-.App-logo {
-  height: 40vmin;
+.app-header {
+  height: 10%;
+  background-color: #1a1b1b;
+  display: flex;
+  margin-bottom: 5px;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.heading {
+  font-size: xx-large;
+  font-style: oblique;
+  font-family: cursive;
+  color: rgb(32, 127, 204);
   display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  margin: 20px;
+  margin-top: 40px;
+}
+
+.app-header img {
+  max-height: 100px;
+} 
+
+.media-player {
+  width: 70%;
 }
 
-.App-link {
-  color: #09d3ac;
+.media-player img {
+  max-width: 100%;
 }
+
+.movie-list {
+  float: right;
+  width: 30%;
+}
+
+.movie-box {
+  display: flex;
+}
+
+.movie-box img {
+  max-width: 150px;
+  max-height: 150px;
+  margin: 10px;
+}
+
+.movie-box div {
+    margin: 10px;
+    padding: 5px;
+}
+
+.movie-box h4 {
+  margin: 5px 10px 5px 0px;
+}
+
+.movie-box p {
+  color: grey;
+}
+
+.filter-button {
+  display: flex;
+  margin: 5px
+}
+
+#reset-movies {
```

Comment:
good to play around with other css selectors but in one convention `ids` are generally reserved for js event binding and other DOM manipulation in old architectures of Single Page Applications (SPAs) while class selectors are used for css.

And by old architectures of Single Page Applications, I meant backbone.js, Jquery, handlebars, etc.

Reply:
@MehwishDogar this change was also not done. use class instead of id. one more problem with id is that you can't apply same id to more than one dom node. so what ever the intention is behind using `reset-movies` id. just change this to class.